gapic/utils/lines.py

```diff
@@ -40,6 +40,47 @@ def sort_lines(text: str, dedupe: bool = True) -> str:
     return f'{leading}{answer}{trailing}'
 
 
+def get_subsequent_line_indentation_level(list_item: str) -> int:
+    """
+    Given a list item return the indentation level for subsequent lines.
+    For example, if it is a numbered list, the indentation level should be 3
+    as shown below.
+
+    Here subsequent lines should be indented by 2
+
+    - The quick brown fox jumps over the lazy dog. The quick brown fox jumps
+      over the lazy dog
+
+    Here subsequent lines should be indented by 2
+
+    + The quick brown fox jumps over the lazy dog. The quick brown fox jumps
+      over the lazy dog
+
+    Here subsequent lines should be indented by 3
+
+    1. The quick brown fox jumps over the lazy dog. The quick brown fox jumps
+       over the lazy dog
+
+    """
+    if len(list_item) >= 2 and list_item[0:2] in ['- ', '+ ']:
+        indentation_level = 2
+    elif len(list_item) >= 3 and list_item[0].isdigit() and list_item[1:3] == '. ':
```

Comment:
This isn't robust if we get into double digits. How about searching that anything preceding the first space is a series of digits followed by a period?
```suggestion
    elif re.match("^\d+\. ", list_item):
```